README.md

```diff
@@ -25,6 +25,7 @@ compatibility list:
 
 | wasmd | cosmwasm-vm | cosmwasm-std |
 | ----- | ----------- | ------------ |
+| 0.17  | 0.15        | 0.15         |
```

Comment:
This is only compatible with the *unreleased* 0.18. We will likely support cosmwasm 0.16 with wasmd 0.18, so I would avoid adding this until we make a release.

Leave the PR open, so we don't forget before the release

Reply:
Oh alright!

Reply:
```suggestion
| 0.18  | 0.15-0.16   | 0.15-0.16    |
```